<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Project1" duration-ms="85035" started-at="2020-07-14T11:40:58Z" finished-at="2020-07-14T11:42:23Z">
    <groups>
    </groups>
    <test name="Login Test" duration-ms="8666" started-at="2020-07-14T11:40:58Z" finished-at="2020-07-14T11:41:06Z">
      <class name="com.happy.testcases.LoginTest">
        <test-method status="FAIL" signature="loginTest(java.util.Hashtable)[pri:0, instance:com.happy.testcases.LoginTest@3fffff43]" name="loginTest" duration-ms="7799" started-at="2020-07-14T17:10:58Z" data-provider="dp" finished-at="2020-07-14T17:11:06Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=himanshu@mg, runmode=Y, username=hmnshgrvr@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed due to login issue]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed due to login issue
	at org.testng.Assert.fail(Assert.java:93)
	at com.happy.testcases.LoginTest.loginTest(LoginTest.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- com.happy.testcases.LoginTest -->
    </test> <!-- Login Test -->
    <test name="Create Account Test" duration-ms="74902" started-at="2020-07-14T11:41:08Z" finished-at="2020-07-14T11:42:23Z">
      <class name="com.happy.testcases.CreateAccountTest">
        <test-method status="PASS" signature="createAccountTest(java.util.Hashtable)[pri:0, instance:com.happy.testcases.CreateAccountTest@58ebfd03]" name="createAccountTest" duration-ms="74894" started-at="2020-07-14T17:11:08Z" data-provider="dp" finished-at="2020-07-14T17:12:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{accountname=Himanshu}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createAccountTest -->
      </class> <!-- com.happy.testcases.CreateAccountTest -->
      <class name="com.happy.testcases.LoginTest">
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.happy.testcases.LoginTest@3fffff43]" name="tearDown" is-config="true" duration-ms="1102" started-at="2020-07-14T17:12:23Z" finished-at="2020-07-14T17:12:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.happy.testcases.LoginTest -->
    </test> <!-- Create Account Test -->
  </suite> <!-- Project1 -->
</testng-results>
